eclipse-scout-core/src/id/IdParser.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import {Id, InitModelOf, ObjectType, ObjectWithType, scout, strings} from '../index';
+
+const registry: Map<string, ObjectType<Id<any>>> = new Map();
+
+export class IdParser implements ObjectWithType {
+
+  static ID_TYPENAME_DELIMITER = ':';
```

Comment:
What does this constant have the ID_ prefix, but the other two don't? Maybe this should be called TYPE_NAME_DELIMITER?

Reply:
Agree. Fixed.